Cargo.toml

```diff
@@ -32,13 +33,13 @@ pallet-registrar-runtime-api = { path = "pallets/registrar/rpc/runtime-api", def
 
 container-chain-template-runtime = { path = "container-chain-template/runtime", default-features = false }
 pallet-cc-authorities-noting = { path = "container-chains/pallets/authorities-noting", default-features = false }
+tp-authorities-noting-inherent = { path = "container-chains/primitives/authorities-noting-inherent", default-features = false }
```

Comment:
I would then rename this as ccp-authorities-noting-inherent, to make it consistent with the naming conventions we are following